src/main/io/serial_cli.c

```diff
@@ -588,29 +588,42 @@ static void cliRxFail(char *cmdline)
 
             uint16_t value;
             rxFailsafeChannelMode_e mode = channelFailsafeConfiguration->mode;
-            bool requireValue = channelFailsafeConfiguration->mode == RX_FAILSAFE_MODE_SET;
+            bool requireValue = ((mode == RX_FAILSAFE_MODE_AUTO_OR_SET) && (channel >= NON_AUX_CHANNEL_COUNT));
```

Comment:
break this out into a variable.

'bool isFlightChannel = channel < NON_AUX_CHANNEL_COUNT'